src/AI/SpenderAI.java

```diff
@@ -39,6 +39,14 @@ public String getName()
     {
       return "Spender";
     }
+
+    @Override
+    public String getDescription()
+    {
+      return
+          "All Spender knows is that money in the bank doesn't help on the field, so it tries to spend all funds as quickly as possible.\n" +
+          "This can sometimes result in building units that are not useful.";
```

Comment:
I might replace "it" with "he", but this is more concise than mine would have been.

Reply:
Okedoke